pkg/virtualfs/symlink.go

```diff
@@ -30,3 +30,7 @@ var _ fs.Symlink = (*inmemorySymlink)(nil)
 func (imsl *inmemorySymlink) Readlink(ctx context.Context) (string, error) {
 	panic("Symlinks not supported")
 }
+
+func (imsl *inmemorySymlink) Resolve(ctx context.Context) (fs.Entry, error) {
+	panic("Resolve for mock implementation is not supported")
```

Comment:
```suggestion
	panic("Resolve not supported")
```
Just to match the Readlink method.